InfoLogger/public/Model.js

```diff
@@ -250,10 +250,34 @@ export default class Model extends Observable {
   handleLocationChange() {
     const q = this.router.params.q;
     if (q) {
-      this.log.filter.fromObject(JSON.parse(q));
+      this.parseLocation(JSON.parse(q));
```

Comment:
Following our discussion, it would be great to have the profile at the same level as the query.